roles/installer/tasks/resources_configuration.yml

```diff
@@ -285,5 +285,7 @@
 
 - name: Verify the resource pod name is populated.
   assert:
-    that: awx_task_pod_name != ''
+    that:
+      - awx_task_pod_name != ''
     fail_msg: "Could not find the tower pod's name."
```

Comment:
Do we want to think about moving away from the tower naming scheme in errors since we changed the pod_names? I will leave this as a single comment since it isn't directly applied to the PR, more just for others to see and think about potentially changing it if there are others.

Reply:
yes, but it doesn't have to prevent this from merging.